person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- Hey, we need to update the getters and setters method here to make it easy to read and access those other entities in the Person class below:

```
attr_reader :id
attr_accessor :name, :age, :books, :rentals
```

Reply:
Thank you. Changes implemented